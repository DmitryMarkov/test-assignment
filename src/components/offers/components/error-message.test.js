import React from 'react'
import { render } from '@testing-library/react'
import ErrorMessage from './error-message'

test('ErrorMessage should render properly', () => {
  const { asFragment, getByText } = render(<ErrorMessage />)

  expect(
    getByText(
      /Sorry, we cannot load offers\. If you think this is an error try to reload this page again/
    )
  ).toBeInTheDocument()
  expect(asFragment()).toMatchSnapshot()
})
