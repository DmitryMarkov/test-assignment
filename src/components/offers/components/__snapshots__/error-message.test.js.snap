// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage should render properly 1`] = `
<DocumentFragment>
  <div
    class="css-1wwnm4s"
  >
    Sorry, we cannot load offers. If you think this is an error try to reload this page again.
  </div>
</DocumentFragment>
`;
